docs/docs/intro-graphql.md

```diff
@@ -6,3 +6,169 @@ permalink: intro-graphql/
 
 # Introduction to Mu-Haskell for GraphQL
 
+This document will help you get started in writing your first GraphQL server using Mu-Haskell and `mu-graphql`!
+
+## Using the Stack template
+
+First of all, we've prepared a Stack template to boostrap your project easily, just run:
+
+```sh
+$ stack new your-project https://raw.githubusercontent.com/higherkindness/mu-haskell/master/templates/graphql-server.hsfiles -p "author-email:haskell.curry@47deg.com" -p "author-name:Haskell Curry"
+```
+
+After doing this you'll have a compiling project with all the dependencies in place and a simple `Main.hs` file created.
+
+## Defining your GraphQL Schema
+
+The template will generate a `schema.graphql` file with a basic "hello" query, feel free to change it with your domain types. For the sake of this intro we'll use our beloved library example from the Apollo GraphQL docs:
+
+```graphql
+type Book {
+  title: String!
+  author: Author!
+}
+
+type Author {
+  name: String!
+  books: [Book!]!
+}
+
+type Query {
+  authors: [Author!]!
+  books: [Book!]!
+}
+```
+
+This simple schema defines **three types: a book, an author, and a query**. All GraphQL APIs have a query type which defines all the possible queries you can ask. In our case, we have a query asking for all the authors in the library and another one for all the books, simple. 📚
```

Comment:
Two "simple"s in one line!

[Don't say "simply"](https://docs.google.com/presentation/d/1vlQyI3UcvGVNMuxcOu0J60NiJPvABy_4DkmXZQr-szw/edit#slide=id.g2492c25a48_0_12)